src/textual/app.py

```diff
@@ -500,18 +500,12 @@ def is_headless(self) -> bool:
         """Is the driver running in 'headless' mode?
 
         Headless mode is used when running tests with [run_test][textual.app.App.run_test].
-
         """
         return False if self._driver is None else self._driver.is_headless
 
     @property
     def screen_stack(self) -> Sequence[Screen]:
-        """The current screen stack.
-
-        Returns:
-            A snapshot of the current state of the screen stack.
-
-        """
+        """A snapshot of the current screen stack."""
```

Comment:
Ditto with the `Return:`. I couldn't make the docs link to `Screen` without it. Unless you have found a way of doing that?